packages/features/ee/teams/components/TeamsUpgradeBanner.tsx

```diff
@@ -0,0 +1,38 @@
+import { useRouter } from "next/router";
+
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { trpc } from "@calcom/trpc/react";
+import { TopBanner } from "@calcom/ui/components";
+import { showToast } from "@calcom/ui/v2/core";
+
+export function TeamsUpgradeBanner() {
+  const { t } = useLocale();
+  const router = useRouter();
+  const { data } = trpc.viewer.teams.getUpgradeable.useQuery();
+  const publishTeamMutation = trpc.viewer.teams.publish.useMutation({
+    onSuccess(data) {
+      router.push(data.url);
+    },
+    onError: (error) => {
+      showToast(error.message, "error");
+    },
+  });
+  if (!data) return null;
+  const [team] = data;
+  if (!team) return null;
+  return (
+    <TopBanner
+      text={`Hey! your team "${team.team.name}" needs to be upgraded.`}
```

Comment:
cc @Jaibles @baileypumfleet some better copy would be nice here. We need to make clear that if users don't upgrade let's say in 30 days their teams will be unpublished.

Reply:
Do we have a value for that already? @zomars

Reply:
Should we also add a translation in here?

Reply:
Once we have final copy yes